### ISSUE TITLE
训练好的UIE模型如何使用trustai进行数据增强

### ISSUE THREAD

**Issue by felixstander** [PaddlePaddle/PaddleNLP#4934]
### 请提出你的问题

请提出你的问题
1）版本、环境信息

PaddleNLP 2.5.0
truestai 0.1.12
2）系统环境
linux， Python3.8
3）复现信息
您好，在复现
[applications/text_classification/multi_class/analysis](https://github.com/PaddlePaddle/PaddleNLP/blob/develop/applications/text_classification/multi_class/analysis/dirty.py) 的代码过程，根据README.md要求在Terminal中成功运行了训练&评估代码，如下：
python dirty.py \
    --dataset_dir "../data" \
    --max_seq_length 128 \
    --params_path "../checkpoint/model_best" \
    --batch_size 8 \
    --dirty_num 100 \
    --dirty_file "train_dirty.txt" \
    --rest_file "train_dirty_rest.txt"
<img width="1012" alt="image" src="https://user-images.githubusercontent.com/92704549/220557457-3988ff12-ed2a-43c9-9229-0db5a7baae4a.png">
因为是uie模型，所以最后一层layer名称经过查询是sigmod而非classifier，但是修改后依然报错，报错信息为: ../site_packages/trustai/interpretation/example_level/method/exmaple_base_interpreter.py,line 65,in predict_fn
forward_pre_hook_handle = classifier.register_forward_pre_hook(forward_pre_hook)
AttributeError:'NoneType' object has no attribute "register_forward_pre_hook"
感觉是没有获取的uie模型的layer名称，请问这一块得如何修改？

**Comment by felixstander** [comment]
没人帮忙解答一下吗~

**Comment by zhangyimi** [comment]
你好，trustai暂时只支持分类模型哈。RepresenterPointModel要求模型满足`logits=model(**input)`，其中logits的shape是（B，N)。UIE的输出暂不支持哈。

**Comment by felixstander** [comment]
好的明白，那这方面什么时候会支持？
